custom-vu/src/main/kotlin/jces1209/vu/page/boards/ClassicBoardPage.kt

```diff
@@ -1,27 +1,25 @@
 package jces1209.vu.page.boards
 
-import jces1209.vu.wait
+import jces1209.vu.page.CommonPageLocators
+import jces1209.vu.page.PageLoader
 import org.openqa.selenium.By
 import org.openqa.selenium.WebDriver
-import org.openqa.selenium.support.ui.ExpectedConditions.or
-import org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated
 import java.net.URI
 
 internal class ClassicBoardPage(
     private val driver: WebDriver,
     override val uri: URI
 ) : BoardPage {
-    private val boardLoadedLocators = listOf(
-        By.xpath("//*[contains(text(), 'Your board has too many issues')]"),
-        By.xpath("//*[contains(text(), 'Board not accessible')]"),
-        By.xpath("//*[contains(text(), 'Set a new location for your board')]"),
-        By.cssSelector(".ghx-column")
+
+    private val pageLoader = PageLoader.Builder(
+        webDriver = driver,
+        correctLocators = CommonPageLocators.CLASSIC_BOARDS
```

Comment:
Is it really that common if it's used only once? 😉 
Perhaps it's a good place to inline it.

Reply:
Good point :) Maybe my naming was off, what I meant with this one, was to separate locator definitions from acutal code (dunno if it is a good idea, wdyt?), by moving it somewhere else for example like `CloudPageLocators`, we could have all of locators in one place, and by that we'll have quick overview of whole system in one place. That might be helpful in debuging or onboarding other people into new code.  Just from my perspective I thoughtcould be useful for me when I started adding empty boards. I won't be stubborn though, I also see the value in inling them (encapsulation).

Reply:
> separate locator definitions from acutal code

Locators are the actual code. The idea of [page objects](https://github.com/SeleniumHQ/selenium/wiki/PageObjects) is to encapsulate browser-facing stuff from user-facing stuff. Locators are browser-facing.

> quick overview of whole system in one place

That's a reoccurring illusion. There's never a "whole system" nor a "one place". This code will be reused alongside other page objects, perhaps from a different repo.

Reply:
Ok, got rid of this, now it has `fun cloudErrors()` and all other locators will be specified in pages that want to use this error recognising class